README.mkd

```diff
@@ -318,6 +318,7 @@ T-Head  | XTheadMac       | 2.0.0          | [T-Head ISA extension specification
 T-Head  | XTheadMemPair   | 2.0.0          | [T-Head ISA extension specification](https://github.com/T-head-Semi/thead-extension-spec/releases/download/2.0.0/xthead-2022-09-05-2.0.0.pdf)
 T-Head  | XTheadMemIdx    | 2.0.0          | [T-Head ISA extension specification](https://github.com/T-head-Semi/thead-extension-spec/releases/download/2.0.0/xthead-2022-09-05-2.0.0.pdf)
 T-Head  | XTheadSync      | 2.0.0          | [T-Head ISA extension specification](https://github.com/T-head-Semi/thead-extension-spec/releases/download/2.0.0/xthead-2022-09-05-2.0.0.pdf)
+T-Head  | XTheadVdot      | 2.2.0          | [T-Head ISA extension specification](https://github.com/T-head-Semi/thead-extension-spec/releases/download/2.2.0/xthead-2022-12-04-2.2.0.pdf)
```

Comment:
I don't really mind the version scheme for vendor extension, but 2.2 seems like spec version rather than extension version, and I don't see individual extension version for other T-Head extensions, does it means all other T-Head extensions also bump to 2.2?

Reply:
@kito-cheng From my personal opinion, we should set version of all T-Head ISAs as v1.0, and have individual extension version.
here I copy from @cmuellner's scheme just only, maybe @cmuellner can clarify this, he is our drafter of open-source version scheme :)

Reply:
The version listed here is the version of the document in which the extension is the first time documented.
Since the extensions are stable (in the sense of being implemented in hardware, not in the weaker RVI sense), their version is irrelevant (similar to the version of ratified standard extensions).

I chose the document version to avoid confusion with a "1.0" version that does not match the referenced document. But I don't mind if we switch to "1.0" for all extensions.

For me, it is also ok to change all links to the latest spec version.

Reply:
The table was updated in #28.
So once this PR is updated, this can get merged.